storage/client/queue/src/main/java/com/azure/storage/queue/models/PeekedMessageItem.java

```diff
@@ -14,6 +15,7 @@
  * on a Queue.
  */
 @JacksonXmlRootElement(localName = "QueueMessage")
+@Fluent
 public final class PeekedMessageItem {
```

Comment:
Did we want to change the name of this class to `PeekedMessage` as we did with `DequeuedMessageItem`